### PR TITLE
[Rule Tuning] Misc. DR Rule Tuning

### PR DESCRIPTION
**Pull request by Aegrah**
## Summary
Misc. DR rule tuning.

### CODE REVIEW

**Comment by protectionsmachine** [comment]
## Rule: Tuning - Guidelines

These guidelines serve as a reminder set of considerations when tuning an existing rule.

### Documentation and Context

- [ ] Detailed description of the suggested changes.
- [ ] Provide example JSON data or screenshots.
- [ ] Provide evidence of reducing benign events mistakenly identified as threats (False Positives).
- [ ] Provide evidence of enhancing detection of true threats that were previously missed (False Negatives).
- [ ] Provide evidence of optimizing resource consumption and execution time of detection rules (Performance).
- [ ] Provide evidence of specific environment factors influencing customized rule tuning (Contextual Tuning).
- [ ] Provide evidence of improvements made by modifying sensitivity by changing alert triggering thresholds (Threshold Adjustments).
- [ ] Provide evidence of refining rules to better detect deviations from typical behavior (Behavioral Tuning).
- [ ] Provide evidence of improvements of adjusting rules based on time-based patterns (Temporal Tuning).
- [ ] Provide reasoning of adjusting priority or severity levels of alerts (Severity Tuning).
- [ ] Provide evidence of improving quality integrity of our data used by detection rules (Data Quality).
- [ ] Ensure the tuning includes necessary updates to the release documentation and versioning.

### Rule Metadata Checks

- [ ] `updated_date` matches the date of tuning PR merged.
- [ ] `min_stack_version` should support the widest stack versions.
- [ ] `name` and `description` should be descriptive and not include typos.
- [ ] `query` should be inclusive, not overly exclusive. Review to ensure the original intent of the rule is maintained.

### Testing and Validation

- [ ] Validate that the tuned rule's performance is satisfactory and does not negatively impact the stack.
- [ ] Ensure that the tuned rule has a low false positive rate.

File: rules/_deprecated/impact_potential_linux_ransomware_file_encryption.toml [hunk 1]
```diff
@@ -1,8 +1,9 @@
 [metadata]
 creation_date = "2023/03/20"
+deprecation_date = "2024/07/18"
 integration = ["endpoint"]
-maturity = "production"
-updated_date = "2024/05/21"
+maturity = "deprecated"
```
Comment (w0rk3r) [inline rules/_deprecated/impact_potential_linux_ransomware_file_encryption.toml:5]: Optional: We could do the name change first, and deprecate in the next one

File: rules/linux/command_and_control_suspicious_network_activity_from_unknown_executable.toml [hunk 2]
```diff
@@ -181,46 +181,44 @@ timestamp_override = "event.ingested"
...
-
 [rule.threat.tactic]
 id = "TA0011"
 name = "Command and Control"
 reference = "https://attack.mitre.org/tactics/TA0011/"
 
 [rule.new_terms]
 field = "new_terms_fields"
-value = ["host.id", "process.executable"]
+value = ["process.executable"]
```
Comment (w0rk3r) [inline rules/linux/command_and_control_suspicious_network_activity_from_unknown_executable.toml:220]: Just to be sure, we don't want to check per host, but per env?

**Review by w0rk3r — APPROVED** [review]

File: rules/linux/command_and_control_suspicious_network_activity_from_unknown_executable.toml (hunk 2 above)
Comment (Aegrah) [reply rules/linux/command_and_control_suspicious_network_activity_from_unknown_executable.toml:220]: Correct. It is slightly noisy the way it is right now, and the goal is to capture malware, which is likely not to be present on multiple systems in the environment at the same time, thus using other hosts to reduce noise should be good for this one.

File: rules/_deprecated/impact_potential_linux_ransomware_file_encryption.toml (hunk 1 above)
Comment (Aegrah) [reply rules/_deprecated/impact_potential_linux_ransomware_file_encryption.toml:5]: Was thinking of the same thing, but this rule is bad in terms of performance and FPs, and I got several complaints about it before. So I wanted to get it out of the stack ASAP. We still don't have a guideline for this. Decided on just getting it out in one go, as many customers don't update pre-built rules either.

**Review by shashank-elastic — APPROVED** [review]

**Pull request merged**